opentelemetry-stackdriver/src/lib.rs

```diff
@@ -51,6 +51,9 @@ use yup_oauth2::authenticator::Authenticator;
 #[allow(clippy::derive_partial_eq_without_eq)] // tonic doesn't derive Eq for generated types
 pub mod proto;
 
+#[cfg(feature = "propagator")]
```

Comment:
curious, why this needs to be under feature flag? The user must set the global propagator themselves anyway for this to take effect, right?
(Or feature flag is done to keep extra dependencies as optional? )

Reply:
Yeah, I placed it under the feature flag so that users who don't need it don't unnecessarily get new dependencies in their projects after upgrading to a new library version.